src/Tools/DateTimeComparator.php

```diff
@@ -5,57 +5,36 @@
 class DateTimeComparator
 {
     /**
-     * Get the greatest DateTime between the twice given as parameter
-     * If one DateTime is null, return the other
-     * If the twice DateTime are null, return null
+     * Get the greatest "DateTimeInterface" from the parameters
      */
-    public static function greatest(\DateTimeInterface $dateTime1 = null, \DateTimeInterface $dateTime2 = null)
+    public static function greatest(... $datetimes)
```

Comment:
In order to keep the API consistent you can still apply type hinting, it should work on php5.6 AFAIK.

```php
public static function greatest(\DateTimeInterface ...$datetimes)
```

Reply:
This change breaks possibility to use `null` or non DateTimeInterface argument (see CI). 
- Do we restrict to only DateTimeInterface objects ?
- Do we remove this typehint to keep `null` & non DateTimeInterface usage ? 

IMHO I prefer the second

Reply:
Actually @Awkan you're right, typehint to DateTypeInterface is annoying for the user. It force him to ensure type of every objects. Currently it throw an exception if you don't give only `\DateTimeInterface` instances to the function. 

I update the code